cypress/features/regression/alert.feature

```diff
@@ -54,7 +54,6 @@ Feature: Alert component
       | small        | 380              | sizeSmall       |
       | medium-small | 540              | sizeMediumSmall |
       | medium       | 750              | sizeMedium      |
-      | medium-large | 850              | sizeMediumLarge |
```

Comment:
This size has been removed from the test scenarios here but this size is still specified as a valid value for the size prop in the prop list in the MDX Docs. If this size has been removed correctly, shouldn't it be removed from the MDX too?

Reply:
![Zrzut ekranu 2021-08-19 o 11 31 05](https://user-images.githubusercontent.com/77274590/130045333-db912b53-6c37-4a60-a28b-e9ac34e13e3a.png)

I think it shouldn't be removed. @DlgSHi what do you think?

Reply:
reverted :) good spot